com.github.babluboy.nutty.json

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "com.github.babluboy.nutty",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "nutty",
+    "rename-appdata-file": "com.github.babluboy.nutty.appdata.xml",
+    "finish-args": [
+            "--share=ipc",
+            "--socket=x11",
+            "--device=dri",
+            "--socket=wayland",
+            "--filesystem=home",
+            "--talk-name=ca.desrt.dconf",
+            "--filesystem=xdg-run/dconf",
+            "--filesystem=~/.config/dconf:ro",
+            "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+
+    "build-options": {
+        "cflags": "-O2",
+        "cxxflags": "-O2"
+    },
+    "modules": [
+         {
+                "name": "openjpeg",
+                "buildsystem" : "cmake",
```

Comment:
`cmake-ninja` and add `"config-opts": ["-DCMAKE_BUILD_TYPE=Release"]`, same for all cmake projects.